configs/sites/derecho/packages.yaml

```diff
@@ -24,26 +24,16 @@ packages:
       - libfabric/1.15.2.0
       - cray-pals/1.2.11
     - spec: cray-mpich@8.1.25%gcc@12.2.0 +wrappers
-      prefix: /opt/cray/pe/mpich/8.1.25/ofi/intel/19.0
+      prefix: /opt/cray/pe/mpich/8.1.25/ofi/gnu/9.1
       modules:
       - craype/2.7.20
       - cray-mpich/8.1.25
       - libfabric/1.15.2.0
       - cray-pals/1.2.11
-  python:
-    buildable: False
-    externals:
-    - spec: python@3.10.10+bz2+ctypes+dbm+lzma+nis+pyexpat+pythoncmd+readline+sqlite3+ssl+tix+tkinter+uuid+zlib
-      prefix: /glade/u/apps/derecho/23.06/opt
-  py-pip:
-    buildable: False
-    externals:
-    - spec: py-pip@22.0.4
-      prefix: /usr
 
 ### Modification of common packages
-  #esmf:
-  #  variants: ~xerces ~pnetcdf snapshot=none ~shared +external-parallelio esmf_os=Linux esmf_comm=mpich3
+  esmf:
+    variants: ~xerces ~pnetcdf snapshot=none ~shared +external-parallelio esmf_os=Linux esmf_comm=mpich
```

Comment:
For what it's worth, I _believe_ it is the case that you can just put the unique variants here (esmf_os, esmf_comm), but the rest of the ones from the common config should still be respected. That was my experience on Acorn, anyway.